taca/storage/cli.py

```diff
@@ -16,19 +15,30 @@ def storage(ctx, days, run):
 # Storage subcommands
 @storage.command()
 @click.option('--backend', type=click.Choice(['swestore']), required=True,
-			  help='Long term storage backend')
+              help='Long term storage backend')
 @click.pass_context
 def archive(ctx, backend):
-	""" Archive old runs to SWESTORE
+    """ Archive old runs to SWESTORE
 	"""
-	params = ctx.parent.params
-	if backend == 'swestore':
-		st.archive_to_swestore(days=params.get('days'), run=params.get('run'))
+    params = ctx.parent.params
+    if backend == 'swestore':
+        st.archive_to_swestore(days=params.get('days'), run=params.get('run'))
 
 
 @storage.command()
+@click.option('--site', type=click.Choice(['swestore','archive','illumina','analysis','nas','proc']),
```

Comment:
Even I was concerned about that, was concerned about your preference as you would implement it. But now I know so i will change it :smile: :+1: